src/pmod_ssr.erl

```diff
@@ -72,8 +72,8 @@ off() ->
 init(Slot) ->
     % TODO: expand for more possible Slots
     case {grisp_hw:platform(), Slot} of
-        {grisp_base, gpio1} -> ok;
-        {grisp2, gpio1} -> ok;
+        {_, gpio1} -> ok;
+        {grisp_base, gpio2} -> ok;
         {P, S} -> error({incompatible_slot, P, S})
     end,
     Pin1 = grisp_gpio:open(gpio1_1, #{mode => {output, 0}}),
```

Comment:
If the user uses the Slot `{grisp_base, gpio2}` . You will still open `gpio1_1` for `Pin1` but if I understand the docs correctly, you should open `gpio2_1` instead